server/container/guice/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -153,6 +156,16 @@ JwtConfiguration providesJwtConfiguration(JMAPDraftConfiguration jmapConfigurati
         return jwtPublickeyPemUrl.map(Throwing.function(url -> FileUtils.readFileToString(fileSystem.getFile(url), StandardCharsets.US_ASCII)));
     }
 
+    @Provides
+    @Singleton
+    VersionParser versionParser() {
+        return new VersionParser(supportedVersions());
```

Comment:
Annotate the constructor with @Inject and do the binding normally in `configure()`